doc/nrf/ug_thingy91_gsg.rst

```diff
@@ -0,0 +1,571 @@
+   #. Find the modem firmware zip file with the name similar to mfw_nrf9160_*.zip and the number of the latest version and click :guilabel:`Open`.
+
+      .. note::
+         Do not extract the modem firmware zip file.
+
+   #. Select the zip file and click :guilabel:`Open`.
+   #. In the Programmer application, scroll down in the menu on the left to :guilabel:`Device` and click :guilabel:`Write`.
+
+      .. figure:: /images/programmer_ext_debug_update_modem.png
+         :alt: Programmer - Update modem
+
+         Programmer - Update modem
+
+      The :guilabel:`Modem DFU` window appears.
+   #. In the :guilabel:`Modem DFU` window, click :guilabel:`Write`.
+
+      .. note::
+         If you have issues updating modem firmware, click :guilabel:`Erase all` before trying to update the modem again. In this case, the contents of the flash memory are deleted and the applications must be reprogrammed.
+
+      When the update is complete, a :guilabel:`Completed successfully` message appears.
+
+5. Update the nRF9160 SiP application:
+
+   a. Make sure the SWD selection switch **SW2** is set to nRF91.
+
+   #. In the menu on the left, click :guilabel:`Add file > Browse...`.
+
+      .. figure:: /images/programmer_add_file.png
+         :alt: Programmer - Add file
+
+         Programmer - Add file
+
+      A file explorer window appears.
+
+   #. Navigate to the folder you downloaded and extracted from `Nordic Semiconductor website`_ in the :ref:`Download firmware <download_firmware_thingy91>` step.
+
+   #. Open the folder that contains the HEX files for updating over USB.
+      See the :file:`CONTENTS.txt` file for information on which file you need.
+
+   #. Select the appropriate Asset Tracker v2 firmware file.
+
+.. include:: ug_thingy91_gsg.rst
+   :start-after: programing_ext_dp_start
+   :end-before: programing_ext_dp_end
+
+..
+
+Next, you need to create an nRF Cloud account if you do not have one already.
+
+Creating an nRF Cloud account
+*****************************
+
+You must sign up with `nRF Cloud`_ before you can start using the service.
+
+To create an nRF Cloud account, complete the following steps:
+
+1. Go to `nRF Cloud`_ and click :guilabel:`Register`.
+#. Enter your email address and choose a password, then click :guilabel:`Create Account`.
+   nRF Cloud will send you a verification email.
+#. Copy the 6-digit verification code and paste it into the registration dialog box.
+   If you do not see the verification email, check your junk mail for an email from no-reply@verificationemail.com.
+
+   If you closed the registration dialog box, you can repeat Step 1 and then click on :guilabel:`Already have a code?`.
+   Then enter your email and the verification code.
+
+You can now sign in on `nRF Cloud`_ with your email and the password your chose.
+After signing in, you are taken to the dashboard view that displays your device count and service usage.
```

Comment:
Does this "taken to" sound a bit too concrete? I think the original "directed to" might be better or just "a dashboard is displayed".